src/pudl/etl/__init__.py

```diff
@@ -262,7 +262,14 @@ def load_dataset_settings_from_file(setting_filename: str) -> dict:
         define_asset_job(
             name="etl_full",
             description="This job executes all years of all assets.",
-            config=default_config,
+            config=default_config
```

Comment:
This will make the `etl_full` pull settings from the yaml file. Explicitly defining settings like this also makes the settings appear in the launchpad by default.